microsoft-edge/devtools-guide-chromium/whats-new/2023/01/devtools-109.md

```diff
@@ -0,0 +1,203 @@
+*  The JavaScript that was running on the page at the time of recording.
+*  Snapshots of the DOM.
+
+When saving performance profiles, heap snapshots, allocation timelines, or allocation sampling, you can now export a new `.devtools` file format.  When you then import the `.devtools` file, a new instance of DevTools opens, with the state of your **Elements**, **Console**, and **Sources** tools preserved.  These enhanced traces offer a powerful new way to collaborate and share the information in your DevTools.
+
+To try the enhanced traces feature:
+
+1. In DevTools, click the **Settings** (![Settings icon](../../../media/settings-gear-icon-light-theme.png)) button.
+
+1. In the **Persistence** section of the **Preferences** settings page, select the **Export enhanced performance and memory traces** checkbox:
+
+   ![Settings > Preferences > Persistence checkboxes](./devtools-109-images/enhanced-trace-settings.png)
+
+1. If you want to preserve Console messages, JavaScript, or DOM snapshots, select the corresponding checkboxes.
+
+1. Click the **Close** (**x**) button in DevTools **Settings**.
+
+1. In the **Performance** tool, take a recording.
+
+1. Right-click the **Save profile** (![Save profile icon](./devtools-109-images/save-profile-icon.png)) button, and then select **.devtools (enhanced format for Microsoft Edge)**:
+
+   ![Exporting a trace](./devtools-109-images/enhanced-trace-export.png)
+
+1. In the **Save As** dialog, save the new `.devtools` file.
+
+
+**Importing:**
+
+1. Import the `.devtools` file from within the **Performance** tool by clicking the **Load profile** (![Load profile icon](./devtools-109-images/load-profile-icon.png)) button:
+
+   ![Importing a trace from within the Performance tool](./devtools-109-images/enhanced-trace-import.png)
+
+1. A new DevTools window opens, containing a subset of the tools, including the **Performance** and/or **Memory** tool.  The **Elements**, **Console**, and **Sources** tools are pre-populated with the preserved state:
+
+   ![Trace import console](./devtools-109-images/enhanced-trace-import-console.png)
+
+This is a new feature that the DevTools team is actively experimenting with, so please [send us your feedback in our GitHub repo](https://github.com/MicrosoftEdge/DevTools/issues)!
+
+<!-- todo: link to article when PR is live, after merge main into present branch
+See also:
+* [Share performance and memory traces](../../../evaluate-performance/share-traces.md)
+
+PR that adds that article:
+https://github.com/MicrosoftDocs/edge-developer/pull/2388 "Documentation for the DevTools enhanced traces feature"
+Rendered:
+https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/enhanced-traces/microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md
+-->
+
+
+<!-- ====================================================================== -->
+## Debug long-running Recalculate Styles events in the Performance tool with new selector stats
```

Comment:
I just feel like this title is too long.

```suggestion
## Debug long-running Recalculate Style events with new selector stats
```

Reply:
fixed locally